src/app/workbench/sidebar/WorkbenchUtilsSidebar.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useMemo, useState } from 'react';
+import { useAppSelector } from '../../../hooks/useAppSelector';
+import { DetailsSidebar } from './DetailsSidebar';
+import { IWorkbench } from '../../../store';
+
+export function WorkbenchUtilsSidebar({ workbench, openTab = '' }: { workbench: IWorkbench; openTab?: string }) {
+  const ordino = useAppSelector((state) => state.ordino);
+
+  const [openedTab, setOpenedTab] = useState<string>(openTab);
+
+  const openedTabComponent: React.ReactElement = useMemo(() => {
+    switch (openedTab) {
+      case 'add': {
+        return <div style={{ width: '250px' }}>Adding something</div>;
+      }
+      case 'mapping': {
+        return <DetailsSidebar workbench={workbench} />;
+      }
+      case 'filter': {
+        return <div style={{ width: '250px' }}>Filter something</div>;
+      }
+      case 'comment': {
+        return <div style={{ width: '250px' }}>Comment something</div>;
+      }
+      default: {
+        return <div style={{ width: '250px' }}>Thats weird</div>;
```

Comment:
More speaking "error case", user has no idea what 'that's weird' implies

Reply:
Ya definitely true, all of these hard coded components aren't great, and will be changed as we create actual components for them, but that one's particularly bad. Should probably make a "error" type component to normalize this kind of thing in the future, but changed the text to "There was an error finding the correct tab" for now.